unattended_installer/install_functions/installCommon.sh

```diff
@@ -541,6 +541,8 @@ function installCommon_rollBack() {
 
     common_remove_gpg_key
 
+    eval "systemctl daemon-reload"
```

Comment:
```suggestion
    eval "systemctl daemon-reload ${debug}"
```

Please redirect the output to the logfile by using ${debug}.

Reply:
Added.